config/charts/inferencepool/README.md

```diff
@@ -45,6 +45,25 @@ Then apply it with:
 $ helm install vllm-llama3-8b-instruct ./config/charts/inferencepool -f values.yaml
 ```
 
+### Install with Custom EPP Plugins Configuration
+
+To set custom EPP plugin config, you can pass it as an inline yaml. For example:
+
+```yaml
+  pluginsCustomConfig:
+    custom-plugins.yaml: |
+      apiVersion: inference.networking.x-k8s.io/v1alpha1
+      kind: EndpointPickerConfig
+      plugins:
+      - type: custom-scorer
+        parameters:
+          custom-threshold: 64
+      schedulingProfiles:
+      - name: default
+        plugins:
+        - pluginRef: custom-scorer
+```
```

Comment:
nit: can you add the `inferenceExtension` key as in the other sections?
```suggestion
```yaml
  inferenceExtension:
    pluginsCustomConfig:
      custom-plugins.yaml: |
        apiVersion: inference.networking.x-k8s.io/v1alpha1
        kind: EndpointPickerConfig
        plugins:
        - type: custom-scorer
          parameters:
            custom-threshold: 64
        schedulingProfiles:
        - name: default
          plugins:
         - pluginRef: custom-scorer
```
```